src/base-element.js

```diff
@@ -503,8 +530,8 @@ export class BaseElement {
    * @param {!Element|!Array<!Element>} elements
    * @protected
    */
-  schedulePause(elements) {
-    this.resources_.schedulePause(this.element, elements);
+  scheduleUnload(elements) {
```

Comment:
To clarify, this is AMP carousel (not serpusel) - this would seem like a quite a big thing to unload the whole iframe. This will also make it harder for us to implement additional layers of unloading. E.g. we may decide to later allow unloading of YouTube videos which are iframes, but it's very unlikely that this is a desired behavior for the AMP carousel - i.e. the play position will be lost.